### DISCUSSION TITLE
Please help for font vietnames.

### DISCUSSION THREAD

**Discussion by loitranit** [Kaplas80/TF3#46] [Q&A]
- Game : Yakuza Kiwami 2
- My Characters: àáâãèéêìíòóôõùúýăđĩũơưạảấầẩẫậắằẳẵặẹẻẽếềểễệỉịọỏốồổỗộớờởỡợụủứừửữựỳỵỷỹ
ÀÁÂÃÈÉÊÌÍÒÓÔÕÙÚÝĂĐĨŨƠƯẠẢẤẦẨẪẬẮẰẲẴẶẸẺẼẾỀỂỄỆỈỊỌỎỐỒỔỖỘỚỜỞỠỢỤỦỨỪỬỮỰỲỴỶỸ
- In Yakuza Kiwami 2\data\font\en\yakuza_ex.dds . I see there are Vietnamese characters. Is there a way this can use yakuza_ex.dds instead of yakuza.dds?
- Thank you for helping me.

**Comment by Kaplas80** [comment]
I'm sorry, but I only can help you with the font if your language has an [ISO-8859-XX encoding](https://en.wikipedia.org/wiki/ISO/IEC_8859), so I can replace `yakuza.dds` font.

`yakuza_ex` font has a different format and I don't know how to force the game to use it.
